Entities/AHuman.cpp

```diff
@@ -1906,10 +1900,10 @@ void AHuman::UpdateAI()
         // Calculate the path to the target brain if need for refresh (note updating each pathfindingupdated causes small chug, maybe space em out with a timer?)
         // Also if we're way off form the path, or haven't made progress toward the current waypoint in a while, update the path to see if we can improve
         // Also if we seem to have completed the path to the current waypoint, we should update to get the path to the next waypoint
-        if (m_UpdateMovePath || (m_ProgressTimer.IsPastSimMS(10000) && m_DeviceState != DIGGING) || (m_MovePath.empty() && m_MoveVector.GetLargest() < m_CharHeight * 0.5f))// || (m_MoveVector.GetLargest() > m_CharHeight * 2))// || g_SceneMan.GetScene()->PathFindingUpdated())
+        if (m_UpdateMovePath || (m_ProgressTimer.IsPastSimMS(10000) && m_DeviceState != DIGGING) || (m_MovePath.empty() && m_MoveVector.GetLargest() < m_CharHeight * 0.5F))// || (m_MoveVector.GetLargest() > m_CharHeight * 2))// || g_SceneMan.GetScene()->PathFindingUpdated())
         {
             // Also never update while jumping
-            if (m_DeviceState != JUMPING)
+            if (m_ObstacleState != JUMPING)
```

Comment:
I imagine device state could never be jumping here? Also, the cpp AI stuff doesn't really matter, but whatever.

Reply:
Jumping isn't one of the enum options for device state.
And I know that AI somehow works through lua, but as long as this function is in the source I thought there's no reason not to give it a bugfix.

Reply:
Yeah, that's the problem with weak enums like this haha, compiler won't tell you if you do something weird like this.
And for sure, no harm in fixing it, skeletons still use it I think.